linker/Tests/Mono.Linker.Tests.Cases/CoreLink/DelegateAndMulticastDelegateKeepInstantiatedReqs.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Runtime.Serialization;
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+
+namespace Mono.Linker.Tests.Cases.CoreLink {
+	/// <summary>
+	/// Delegate and is created from 
+	/// </summary>
+	[SetupLinkerCoreAction ("link")]
```

Comment:
This will be problematic as the mscorlib will be used for execution.

Reply:
I don’t follow.  Are you saying that it’s bad that this test is dependent on a particular mscorlib?

Reply:
I am saying it won't be easy to run the test